wikipedia-dark.user.css

```diff
@@ -184,7 +184,7 @@ domain("wiki.rpcs3.net") {
   h2[style*="background:#efefef" i], tr[bgcolor*="#eee" i],
   tr[style*="background-color:#eee" i],
   .wikitable.charts-zebragrey > tbody > tr:nth-of-type(even),
-  tr[style*="background:#fbf8d6" i], table.flextable td {
+  tr[style*="background:#fbf8d6" i], table.flextable td, table.nmbox td {
```

Comment:
What about including `table.nmbox th` for the header?

Reply:
Sure thing! Just a doubt (I'm a little rusty with HTML), is a need to mention `table.nmbox tr` if both `table.nmbox th` and `table.nmbox td` are explicitly mentioned?

Reply:
No, you don't need to include the `tr`. The `th` and `td` styles cover it up.

Check out this example - http://jsfiddle.net/Mottie/uqh9znLk/

Reply:
Good to know, thanks! Will drop `tr` and keep only only `th` and `td`